### PR TITLE
Update form uploder styles

### PR DESCRIPTION
**Pull request by 7alip**
Updated styles and handling copy url

### CODE REVIEW

File: packages/ui/src/components/FormUploader/MenuImageItem.tsx [hunk 1]
```diff
@@ -1,79 +1,121 @@
...
+type MenuImageButtonProps = {
+  url: string
+  children: ReactNode
+}
+
+const MenuImageButton: FC<MenuImageButtonProps> = ({ url, children }) => {
+  const { onCopy, setValue, hasCopied } = useClipboard('')
+
+  useEffect(() => {
+    // TODO: Change it to link if it's not an image
```
Comment (7alip) [inline packages/ui/src/components/FormUploader/MenuImageItem.tsx:35]: We can link to any file

File: packages/ui/src/components/FormUploader/MenuImageItem.tsx [hunk 2]
```diff
@@ -1,79 +1,121 @@
...
+    </Button>
+  )
+}
+
+export const MenuImageItem: FC<MenuImageItemProps> = ({ image, onDelete }) => {
+  const entries = Object.entries(image.formats ?? {})
+
+  return (
+    <Box pos={'relative'}>
+      {/* TODO: Display pdf icon for pdfs, File icon for other file types */}
```
Comment (7alip) [inline packages/ui/src/components/FormUploader/MenuImageItem.tsx:70]: It's better to display an icon instead of a broken image

**Review by 7alip — COMMENTED** [review]
@myniqx Can you check this

**Review by myniqx — APPROVED** [review]

File: packages/ui/src/components/FormUploader/MenuImageItem.tsx (hunk 2 above)
Comment (myniqx) [reply packages/ui/src/components/FormUploader/MenuImageItem.tsx:70]: i ll add those things after merge.

**Pull request merged**